static/app/views/alerts/rules/metric/ruleForm.tsx

```diff
@@ -1108,6 +1182,9 @@ class RuleFormContainer extends DeprecatedAsyncComponent<Props, State> {
               onTimeWindowChange={value => this.handleFieldChange('timeWindow', value)}
               disableProjectSelector={disableProjectSelector}
               isExtrapolatedChartData={isExtrapolatedChartData}
+              monitorType={monitorType}
+              activationCondition={activationCondition}
+              onMonitorTypeSelect={this.handleMonitorTypeSelect}
```

Comment:
unrelated to this change
--
we should make a tech debt item to completely refactor this form... 1200 lines and there's a lot of improvements we could do to this, from a technical standpoint alone.

Reply:
Oh yea absolutely.

I hate everything about this form :|